packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.mdx

```diff
@@ -119,35 +119,52 @@ class FancyFocus extends FocusManager {
-| items         | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | alias for `children`                                                                                                         |
-| wrapSelected  | boolean                                                 | false    | enables wrapping behavior, so `selectNext` selects the first item if the current item is the last on the list and vice versa |
+| name         | type            | default | description                                                        |
+| ------------ | --------------- | ------- | ------------------------------------------------------------------ |
+| direction    | 'column'\|'row' | false   | Determines the key handling for focus ('left/right' or 'up/down'). |
+| wrapSelected | boolean         | false   | Enables wrapping behavior for focus navigation.                    |
 
 ### Methods
 
-#### appendItems
+#### appendItems(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>, idx: number): void
 
 Appends items to child list and updates focus
 
 ##### Arguments
 
-| name  | type                                                    | required | default | description     |
-| ----- | ------------------------------------------------------- | -------- | ------- | --------------- |
-| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | items to append |
+| name  | type                                                    | required | default | description                   |
+| ----- | ------------------------------------------------------- | -------- | ------- | ----------------------------- |
+| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | The list of items to append   |
+| idx   | number                                                  | true     | []      | The index to insert the items |
 
-#### render
+#### prependItems(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>): void
+
+Prepends items to the child list and updates the focus
+
+| name  | type                                                    | required | default | description                 |
+| ----- | ------------------------------------------------------- | -------- | ------- | --------------------------- |
+| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | The list of items to append |
+
+#### removeItemAt(index: number): void
+
+Removes an item at a specific index from the child list and updates the focus
+
+#### render(): void
 
 A no-op function that is called when `selectedIndex` is set. Can be overridden by classes that extend `FocusManager` for custom render behavior.
 
-#### selectPrevious
+#### selected():<DocsLink id="lng.Component">lng.Component</DocsLink>
```

Comment:
A question for @erautenberg, do we include accessors in the documentation?

Reply:
What do guys think about this? @erautenberg @jazzyw

Reply:
We do document accessors in the TypeScript documentation. In my opinion it could be helpful to add accessors in the component docs. 

This addition could also be a separate ticket as all the current docs files do not document accessors and will have to be updated.